packages/flutter/lib/src/widgets/drag_target.dart

```diff
@@ -98,6 +98,64 @@ enum DragAnchor {
 /// widget always displays [child].
 ///
 /// {@youtube 560 315 https://www.youtube.com/watch?v=QzA4c4QHZCY}
+/// 
+/// {@tool dartpad --template=stateless_widget_material}
```

Comment:
This will need to be a stateful widget since you need to keep track of your `acceptedData`. Otherwise, it will keep resetting to 0 in your sample.
```suggestion
/// {@tool dartpad --template=stateful_widget_material}
```

Reply:
For more info on how the templates look: https://github.com/flutter/flutter/blob/master/dev/snippets/config/templates/README.md#materialapp-templates